src/main/java/seedu/address/model/plan/PlanDateTime.java

```diff
@@ -12,11 +12,13 @@
  * Guarantees: immutable; is valid as declared in {@link #isValidDateTime(String)}
  */
 public class PlanDateTime {
-    public static final String MESSAGE_CONSTRAINTS =
-            "Date Time should be of the form YYYY-MM-DD-HH:MM";
+    public static final String MESSAGE_INVALID_DATETIME = "Date-Time given is invalid.\n";
 
-    public static final String FUTURE_MESSAGE_CONSTRAINT =
-            "Date Time should be in the future!";
+    public static final String MESSAGE_CONSTRAINTS = MESSAGE_INVALID_DATETIME + " "
+            + "Date Time must be in YYYY-MM-DD-HH:MM format";
```

Comment:
Inconsistent with what you wrote in the UG, which was "Date-Time given is invalid. Ensure that the Date-Time provided is not in the past."